docs/Mainpage.md

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
My goal was for all the doxygen docs to use the same file name: user_guide.md.

So can we keep that name and keep that as our main page for this repo as well?

Reply:
Yes this is doable, but it will be misleading because now the main page would be called user_guide.md, and there would be a separate md file which now contains the actual user guide, so what should we call that?  In other words, right now we have the following in nceplibs-bufr/docs:

`Mainpage.md` - the main page with overall library description
`ReleaseNotes.md` - the release notes for each version
`UserGuide.md` - the actual user guide; right now this is just a link to the HTML docs but will eventually contain the reformulated markdown content

So again, if we now rename Mainpage.md as user_guide.md, then what do we call UserGuide.md?  I would actually argue that it makes more sense to have a common Mainpage.md (or we can call it main_page.md or whatever you want) in every repo, and if you want we can also require a separate release notes file and separate user guide file.  But the point is I don't think we should dictate that the user guide has to be the main page in every repo.

My $0.02 anyway.

Reply:
I think we should dictate that the user_guide.md is the main page. I don't see any benefit of having different names in different project. ;-)

So have a file called user_guide.md which is the doxygen mainpage.

I set them all up the same way to make it easy to work on different projects.

Reply:
Understood, and again I can do that, but the question still remains: if I want to have the actual User Guide documentation in a different file, then what should I call that file if the name `user_guide.md` is already taken as the name for the main page?  Suggestions?  I can certainly make up any filename I want, but is this something you'd also want to standardize across all projects?

It will just be a bit confusing to have the main page named as `user_guide.md` when the User Guide itself is located elsewhere in a file with a different name.